test/unit/artefact/artefact_test.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../../test_helper'
+
+class Artefact::ArtefactTest < ActiveSupport::TestCase
+
+  context "taggable?" do
+
+    should "return true for artifacts not owned by publisher, smartanswers or testapp" do
+      Settings.expects(:untaggable_apps).returns(%w{ publisher smartanswers testapp }).at_least(1)
```

Comment:
refactored to use config/settings/test.yml for most tests, and just mocking the result of Settings.untaggable_apps for the one test that needs a different setting.